.github/workflows/test-cloudflarekv.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches:
+      - master
+      - main
+    paths:
+      - "cloudflarekv/**"
+  pull_request:
+    paths:
+      - "cloudflarekv/**"
+name: "Tests Cloudflare KV"
+jobs:
+  Tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version:
+          - 1.21.x
+          - 1.22.x
+    steps:
+      - name: Fetch Repository
+        uses: actions/checkout@v4
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "${{ matrix.go-version }}"
+      - name: Run Test
```

Comment:
How do we run this tests without a Cloudflare account? What is `cloudflare-go` doing to Mock these calls?

Reply:
Well I saw an test account at https://github.com/cloudflare/cloudflare-go/blob/master/consts.go#L22.

I'll ask that I can to use it or is there any other ways to test

Reply:
Sounds good, open a ticket there and mention this PR. Maybe that's the way of really testing this.

Reply:
Since there is no issue template of making questions, I mailed about this topic. I'll tell you if there is a good solution.

Or if the communication seems to be longer, I'll add Mock Server of `cloudflare.API`

Reply:
@Geun-Oh Fill it as a Bug Report, since this falls under "Not documented". They aint going to answer an email.

Reply:
Okay. I got it.

Reply:
@gaby As he said on [https://github.com/cloudflare/cloudflare-go/issues/1605#issuecomment-2015780675](https://github.com/cloudflare/cloudflare-go/issues/1605#issuecomment-2015780675), I'll make mock server for it.

Let me know that I can write mock code in my code bundle or dockerize it.

Reply:
@Geun-Oh Did some googling and this exists https://github.com/cloudflare/miniflare

It runs a fake cloudflare for testing. The GitHub runnr already has nodejs18 installed, we just need to npm install miniflare and run to before the test.

Probably worth testing locally first with miniflare installed locally

Reply:
@gaby Thanks for searching it. I'll take it and tell you if there is an update